app/views/advanced_messenger/_html_head.html.erb

```diff
@@ -0,0 +1,15 @@
+<% content_for :header_tags do %>
+    <%= stylesheet_link_tag 'advanced_messenger', :plugin => 'redmine_advanced_messenger' %>
+<% end %>  
+<%= javascript_tag do%>
+
+    function renderAllUnreadNotificationsIndicator(unreadNotificationsCount) {
```

Comment:
I think it might be ok to moved all independent js functions in the new js file mentioned above.

This function should, probably, receive another paramter to compensate for `<%=t(:top_app_menu_unread_notifications)%>` not rendering inside js files.

Reply:
Because I moved the functions in another partial that is executed only once, the constructions like ```<%=t(:top_app_menu_unread_notifications)%>``` are still working